clusterloader2/pkg/measurement/util/runtimeobjects/runtimeobjects_test.go

```diff
@@ -538,12 +538,17 @@ func TestGetReplicasFromRuntimeObject(t *testing.T) {
 		if err := scheme.Scheme.Convert(obj, unstructured, nil); err != nil {
 			t.Fatalf("error converting controller to unstructured: %v", err)
 		}
-		replicas, err := runtimeobjects.GetReplicasFromRuntimeObject(fakeClient, unstructured)
+		replicasWatcher, err := runtimeobjects.GetReplicasFromRuntimeObject(fakeClient, unstructured)
 		if err != nil {
 			t.Fatalf("get replicas from runtime object failed: %v", err)
 		}
 
-		if expected[i] != replicas {
+		replicas, err := runtimeobjects.GetReplicasOnce(replicasWatcher)
```

Comment:
don't we have to call Start() before calling GetReplicasOnce?

Reply:
GetReplicasOnce does it internally